.ci/bears.prune.sh

```diff
@@ -0,0 +1,7 @@
+# Delete empty bear directories
+
+for dir in bears/*/; do
+  if [[ -z "$(ls $dir/ 2>/dev/null | egrep -v '(__|.xml)')" ]]; then
```

Comment:
So this looks for directories without any Bear's in it..

The `__` is for `__init__` and `__pycache__`
`.xml` is because of `bears/scala/scalastyle_config.xml`

I think a much better approach would be to look for one file matching `*Bear.py`